pkg/proxy/iptables/proxier.go

```diff
@@ -758,6 +758,11 @@ func (proxier *Proxier) syncProxyRules() {
 		klog.V(4).Infof("syncProxyRules took %v", time.Since(start))
 	}()
 
+	localAddrs, err := utilproxy.GetLocalAddrs()
+	if err != nil {
+		klog.Errorf("Failed to get local addresses during proxy sync: %v, assuming external IPs are not local", err)
+	}
```

Comment:
Should we log if len(localAddrs) == 0 ?  You specifically tested for it below, but I think that would be a nice thing to log.

Reply:
done